### PR TITLE
Fixed #25104 -- Added httpd.conf hyperlink

### PR DESCRIPTION
**Pull request by za**
Add httpd.conf hyperlink to wiki.apache.org
Add newlines so that comply with 79 chars maximum.

### CODE REVIEW

**Comment by timgraham** [comment]
That's a nice improvement, but for beginners it could still be a bit confusing on platforms where the Apache config isn't named `httpd.conf`. Also, I think it'd be better to point people toward configuration a virtual host rather than editing the main configuration file. What do you think?

File: docs/howto/deployment/wsgi/modwsgi.txt [hunk 1]
```diff
@@ -25,9 +25,11 @@ Basic configuration
...
 
 Once you've got mod_wsgi installed and activated, edit your Apache server's
-``httpd.conf`` file and add the following. If you are using a version of Apache
-older than 2.4, replace ``Require all granted`` with ``Allow from all`` and
-also add the line ``Order deny,allow`` above it.
+``httpd.conf``_ file and add the following. If you are using a version of
+Apache older than 2.4, replace ``Require all granted`` with ``Allow from all``
+and also add the line ``Order deny,allow`` above it.
+
+.. _httpd.conf: http://wiki.apache.org/httpd/DistrosDefaultLayout
```
Comment (timgraham) [inline docs/howto/deployment/wsgi/modwsgi.txt]: (please use https links where possible)

**Comment by za** [comment]
@timgraham OK, I'll try to do that. In terms of formatting, is it ok, to put the hyperlink in a bold formatted text?

**Comment by timgraham** [comment]
No, you'll need to remove the backticks for the link to work correctly. See https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#getting-started-with-sphinx so you can test yourself.

**Comment by za** [comment]
@timgraham Thanks, I'll have a look on that :sunglasses:

**Comment by za** [comment]
_/me: update django docs locally: success_

**Comment by timgraham** [comment]
merged in d9e150b3117521f7c24dc6393eb29eebc3cc376a , thanks!

**Pull request closed without merging**